src/ray/core_worker/transport/actor_scheduling_queue.cc

```diff
@@ -186,6 +214,19 @@ void ActorSchedulingQueue::ScheduleRequests() {
   }
 }
 
+void ActorSchedulingQueue::ProcessRequest(InboundRequest &&request) {
+  auto task_id = request.TaskID();
+  auto pool = pool_manager_->GetExecutor(request.ConcurrencyGroupName(),
+                                         request.FunctionDescriptor());
+  if (pool == nullptr) {
+    AcceptRequestOrRejectIfCanceled(task_id, request);
+  } else {
+    pool->Post([this, request = std::move(request), task_id]() mutable {
+      AcceptRequestOrRejectIfCanceled(task_id, request);
+    });
+  }
+}
```

Comment:
man the naming is so puzzling... if `request.CanExecute()`, we `ProcessRequest`, which gets an `Executor`, that will then `AcceptRequestOrReject`

IMO let's leave the world a little better than we found it and name this `ExecuteRequest`